cornac/models/dmrl/recom_dmrl.py

```diff
@@ -0,0 +1,440 @@
+# Copyright 2018 The Cornac Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+from typing import List, Tuple
+
+import numpy as np
+import torch
```

Comment:
We don't have torch, PIL, transformers installed by default. Therefore, it will cause import errors when others not using DMRL model. Please move the import statements into `_fit_dmrl()` and `score()` functions accordingly.